packages/odyssey-react-mui/src/Typography.tsx

```diff
@@ -93,8 +93,11 @@ export type TypographyProps = {
   | "ariaLabelledBy"
   | "testId"
   | "translate"
+  | "role"
 >;
 
+type PublicTypographProps = Omit<TypographyProps, "role">;
```

Comment:
We have a pattern of not using `Omit` in Odyssey to avoid props we don't want getting revealed to other components when we're not expecting it. It's a habit that we make things explicit, not implicit.

Is it possible we can change this to `Pick` instead?

Or maybe we should simply change `role` on `Typography` to also be `isPresentational` like the other ones components? I think that'd be fine. There's no reason for us to want to change it anyway as the `role` is gonna be set by MUI unless we explicitly want it presentational.

Reply:
updated to use isPresentational